include/picongpu/plugins/openPMD/WriteSpecies.hpp

```diff
@@ -167,7 +168,6 @@ namespace picongpu
                     mapper,
                     rp.particleFilter);
 
-                rp.dc.releaseData(MallocMCBuffer<DeviceHeap>::getName());
```

Comment:
Could you keep this line and guard it for non-GPU devices `#if(PMACC_CUDA_ENABLED == 1 || ALPAKA_ACC_GPU_HIP_ENABLED == 1)`